.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,14 @@
+---
+name: User Story
+about: Suggest an idea for this project
+title: ''
+labels: research, user-story
```

Comment:
New user stories are automatically given the `research` label, as I assume stories should first at least trigger us to ask "is research needed on this?"

Happy to remove that if we'd rather proactively add that label to stories that are going through research instead.

Reply:
Could we do "research?" — show that it's not definitive?

Reply:
For context, this conversation continued into slack: https://gsa-tts.slack.com/archives/C02K6V0UNE9/p1637600423002900